packages/bot/src/scratch/blocks/Text/text_prompt_ext.js

```diff
@@ -34,7 +32,6 @@ Blockly.Blocks.text_prompt_ext = {
                 },
             ],
             output         : 'String',
-            outputShape    : Blockly.OUTPUT_SHAPE_SQUARE,
```

Comment:
You can change this to

```suggestion
            outputShape    : Blockly.OUTPUT_SHAPE_ROUND,
```

Reply:
done